src/Components/Nav/Nav.css

```diff
@@ -21,3 +21,17 @@
   color: var(--hover);
   background-color: var(--primary-color);
 }
+
+.user-icon-container {
+  position: relative;
+}
+
+.logout-text {
+  top: 35px;
+  position: absolute;
+  font-size: 20px;
```

Comment:
be consistent about unit

Reply:
i would suggest use rem so that your font size remains consistent throughout all the pages